ladybug/skymodel.py

```diff
@@ -224,6 +224,129 @@ def zhang_huang_solar_split(altitudes, doys, cloud_cover, relative_humidity,
     return dir_norm_rad, dif_horiz_rad
 
 
+"""LUMINOUS EFFICACY OF THE SKY"""
+
+
+def estimate_illuminance_from_irradiance(
+        altitude, ghi, dni, dhi, dew_point, rel_airmass=None):
+    """Estimate sky illuminance components from irradiance components.
+
+    This function uses actual zenith rather than apparent zenith.
+
+    Note:
+        [1] Perez R. (1990). 'Modeling Daylight Availability and Irradiance
+        Components from Direct and Global Irradiance'. Solar Energy.
+        Vol. 44. No. 5, pp. 271-289. USA.
+
+    Args:
+        altitude: Solar altitude angle in degrees.
+        ghi: Number for Global Horizontal Irradiance in W/m2.
+        dni: Number for Direct Normal Irradiance in W/m2.
+        dhi: Number for Diffuse Horizontal Irradiance in W/m2.
+        dew_point: Surface dewpoint in degrees C.
+        rel_airmass: A number between 1 and ~38 representing the ratio of air mass
+            between the sun and the observer and the air mass that is directly above
+            the observer. Default is None, which will simply use the input solar
+            altitude and the get_relative_airmass function in this module with
+            the kastenyoung1989 model to compute this value.
+
+    Returns:
+        gh_ill: Value for Global Horizontal Illuminance in lux.
+        dn_ill: Value for Direct Normal Illuminance in lux.
+        dh_ill: Value for Diffuse Horizontal Illuminance in lux.
+        z_lum: Value for Zenith Luminance in lux.
+
+    """
+    if altitude > 0:
```

Comment:
This is totally fine but it might have been easier to read if you would have wrote it the other way around:

```python
if altitude <= 0:
    return 0, 0 ,0, 0
# and the rest of the code goes here with no indentations
# ...
```

Reply:
Good point. Done!